daisy_workflows/export/README.md

```diff
@@ -1,38 +1,77 @@
-## Daisy disk_export workflow
+# Daisy image/disk export workflows
 Exports a GCE disk to a GCS location.
 
+There are two types of export worklfows depending on the required output 
+format. 
+
+## GCE raw disk image
+`image_export.wf.json` and `disk_export.wf.json` export a raw image in
+a tar.gz, this is the 'native' GCE format and the resulting image can 
+be imported directly to GCE.
+
 Required vars:
-+ `source_disk` GCE disk to export
++ `source_image` GCE image to export
```

Comment:
Looks like you repurposed this disk export workflow documentation to fit image export. Do we have documentation for disk export anywhere now?

Reply:
No, I figured the image export was more clear, and it looks very similar to disk export. We can add disk export at a later time but it would require some additional comments about ensuring disks exist in the same zone/project